easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.4-GCC-11.3.0.eb

```diff
@@ -36,7 +36,16 @@ dependencies = [
     ('UCC', '1.0.0'),
 ]
 
-preconfigopts = './autogen.pl --force && '
+preconfigopts = ' && '.join([
```

Comment:
@Flamefire @bartoldeman I think it makes sense to add a comment block above this line to explain i) why this is needed, ii) why we're not using `autogen.pl`, iii) why the commands we are using are sufficient.

Without that, people are bound to start wondering in the future why we're doing it like this rather than just running `autogen.pl`...

Reply:
Just FYI I did a diff between the result for both methods and the only differences are updates to various autoconf-supplied files which are irrelevant to the changes in the patches. So it's safe.

Reply:
Added a comment